Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build stage
+ARG PYTHON_VERSION=3.8-slim
+FROM python:${PYTHON_VERSION} AS build
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY . /app
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Run stage
+FROM build AS runtime
+
+EXPOSE 8080
+
+CMD ["python", "manage.py", "runserver", "0.0.0.0:8080"]
```

Comment:
The `CMD` instruction only runs the Django server. You need to include the migration command as well to ensure the database is set up correctly before starting the server. Consider combining the migration command with the server start command in a single `CMD` instruction or using a script to execute both commands.